### DISCUSSION TITLE
Favicon in config file cause issues in routing

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#4090] [Help]
Filament config favicon causing issues 
` Illuminate\Routing\UrlGenerator::__construct(): Argument #2 ($request) must be of type Illuminate\Http\Request, null given, `  

I did this then, it cause an issues error above.
```php
    /*
    |--------------------------------------------------------------------------
    | Favicon
    |--------------------------------------------------------------------------
    |
    | This is the path to the favicon used for pages in the admin panel.
    |
    */

    'favicon' => asset('favicon.png'),
```

**Comment by danharrin** [comment]
I don't think Laravel allows you to use the `asset()` helper in config files. You probably need to use '/favicon.png' instead.
